conf/modules/extra_dl.xml

```diff
@@ -10,6 +10,7 @@
     <file name="extra_pprz_dl.h"/>
   </header>
   <init fun="extra_pprz_dl_init()"/>
+  <periodic fun="extra_pprz_dl_periodic()" freq="PERIODIC_FREQUENCY" autorun="TRUE"/>
```

Comment:
hm.... but that means that the periods specified in the telemetry file won't be correct...